docusaurus/docs/reference/config-types/host.v1.md

```diff
@@ -0,0 +1,263 @@
+---
+sidebar_label: host.v1
+sidebar_position: 15
+---
+
+# The `host.v1` Config Type
+
+The `host.v1` configuration type defines how edge routers or tunnelers can make outgoing connections
+for associated services. The config type also allows specifying health checks for the service. An
+endpoint in a `host. v1` config can be specified explicitly or can be configured to use information
```

Comment:
stray space in `host. v1` and `host. v2` appears often (but not always) in this doc.

Reply:
fixed